### PR TITLE
Create ECR repository for Formbuilder Product Page

### PR DESCRIPTION
**Pull request by theKHutDeveloper**

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "formbuilder_ecr_credentials" {
```
Comment (solidgoldpig) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf]: In `formbuilder-repos`, we're following the naming convention, `ecr-repo-fb-{app}`

	module "ecr-repo-formbuilder-product-page" ?
        module "ecr-repo-fb-product-page" ?

`formbuilder_ecr_credentials` sounds like a repo for all our credentials

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.0"
+  repo_name = "formBuilder-product-page"
+  team_name = "formbuilder"
+}
+
+resource "kubernetes_secret" "formbuilder_ecr_credentials" {
+  metadata {
+    name      = "formbuilder_ecr_credentials_output"
```
Comment (jasonBirchall) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf]: Unfortunately, Terraform doesn't like the _ in your secret name. Can you please change?
```
kubernetes_secret.formbuilder_ecr_credentials: metadata.0.name a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
```

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf (hunk 1 above)
Comment (theKHutDeveloper) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf]: Changed the module name

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf (hunk 2 above)
Comment (theKHutDeveloper) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-product-page-prod/resources/ecr.tf]: Changed the metadata name

**Comment by vijay-veeranki-moj** [comment]
Hi Andrien,
Applying changes got failed as you used capital letter for the repo name, can you please change it and raise a PR please. 

repo_name = "formBuilder-product-page"

**Pull request merged**